### DISCUSSION TITLE
Ring peephole knock detection

### DISCUSSION THREAD

**Discussion by galoiring** [dgreif/ring#797] [Q&A] [answered]
hi !
just installed the ring peephole camera and the ring home bridge add-on and its really great !

just one question, is there any way to add the knock detection similar to button press ?
so I can make the automation base on knock or button press ?

that will be great (the ring app is push notify me when someone is knock on the door )
I will be happy to help ..
Gal

**Comment by dgreif** [comment, marked as answer]
This is something I plan to add whenever I get around to #514

**Reply by galoiring** [reply]
I see..
if you can do this it will be great ..(since I see it was added to the list a year ago ..)
